test_validator.py

```diff
@@ -93,6 +102,35 @@ def test_validate_phone_isbn(self):
         self.assertEqual(validator.validate(new_data), False)
         self.assertEqual(1, len(validator.errors.get('phone')))
 
+    def test_validate_invalid(self):
+        validator = Validator()
+        validator.add({"code": [Isbn()]})
+        validator.add({"phone": [Length(min=3, max=13)]})
+        data = {
+            "code": "111",
+            "phone": "+"
+        }
+        result = validator.validate(data)
+        self.assertEqual(False, result)
+        self.assertEqual('Wrong phone', validator.errors.get('phone').pop(0))

```

Comment:
Why here the Error which costumer got is 'Wrong phone' if phone rule wasn't mentioned in rules. Isn't it just Wrong length?